### PR TITLE
StakingV4 bugfixes

### PR DESCRIPTION
**Pull request by mariusmihaic**
## Reasoning behind the pull request
- Delete unused delegation ticker(from obsolete liquid staking)
- Treat overflow cases when computing number qualified of nodes
- Set max number of iterations cap in soft auction config
- Moved `SoftAuctionConfig` from `config.toml` to `systemSmartContractsConfig.toml`

### CODE REVIEW

File: epochStart/metachain/auctionListSelector.go [hunk 1]
```diff
@@ -102,18 +103,50 @@ func getAuctionConfig(softAuctionConfig config.SoftAuctionConfig, denomination i
 		)
 	}
 
+	if minTopUp.Cmp(maxTopUp) > 0 {
```
Comment (iulianpascalau) [inline epochStart/metachain/auctionListSelector.go:106]: :+1:

File: vm/systemSmartContracts/esdt.go [hunk 2]
```diff
@@ -623,10 +616,6 @@ func (e *esdt) createNewToken(
 	if !isTokenNameHumanReadable(tokenName) {
 		return nil, nil, vm.ErrTokenNameNotHumanReadable
 	}
-	if !isTickerValid(tickerName) {
```
Comment (iulianpascalau) [inline vm/systemSmartContracts/esdt.go:626]: I guess we still need this.

File: epochStart/metachain/auctionListSelector.go (hunk 1 above)
Comment (mariusmihaic) [reply epochStart/metachain/auctionListSelector.go:106]: :pray:

File: vm/systemSmartContracts/esdt.go (hunk 2 above)
Comment (mariusmihaic) [reply vm/systemSmartContracts/esdt.go]: Right, very good catch, fixed it

File: epochStart/metachain/auctionListSelector.go [hunk 3]
```diff
@@ -124,7 +124,15 @@ func getAuctionConfig(softAuctionConfig config.SoftAuctionConfig, denomination i
 		)
 	}
 
-	denominator := big.NewInt(int64(math.Pow10(denomination)))
```
Comment (iulianpascalau) [inline epochStart/metachain/auctionListSelector.go]: yup, this will overflow at ~19 decimals

File: vm/systemSmartContracts/esdt_test.go [hunk 4]
```diff
@@ -4032,6 +4032,12 @@ func TestEsdt_ExecuteIssueMetaESDT(t *testing.T) {
 	assert.Equal(t, vmcommon.UserError, output)
 	assert.True(t, strings.Contains(eei.returnMessage, "invalid number of decimals"))
 
+	vmInput.Arguments = [][]byte{[]byte("tokenName"), []byte("ticker"), big.NewInt(10).Bytes()}
```
Comment (iulianpascalau) [inline vm/systemSmartContracts/esdt_test.go:4035]: :+1:

**Review by iulianpascalau — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Pull request merged**